apps/notifications/package.json

```diff
@@ -17,7 +17,8 @@
 	},
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
-		"clean": "npx rimraf dist",
+		"analyze-bundles": "EMIT_STATS=true yarn run build && webpack-bundle-analyzer stats.json dist -h 127.0.0.1 -p 9898 -s gzip",
```

Comment:
If we drop the `-p 9898` it will use port 8888 by default (that's ok I guess?), and then the developer could use `yarn analyze-bundles -p 9898` if they want to change it. Very useful when you want to run two separate `webpack-bundle-analyzer` for different branches.

Reply:
Sounds good, thank you @scinos! 👍